airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -118,8 +122,8 @@ def __init__(self, subdomain: str, start_date: str, ignore_pagination: bool = Fa
         self._ignore_pagination = ignore_pagination
 
     @property
-    def availability_strategy(self) -> Optional["AvailabilityStrategy"]:
-        return None
+    def availability_strategy(self) -> Optional[AvailabilityStrategy]:
+        return HttpAvailabilityStrategy()
```

Comment:
Just wondering, why not define all streams availabilitystrategy as the `ZendeskSupportAvailabilityStrategy` here instead of in `SourceZendeskSupportStream`?

Reply:
We have two types of streams. One of them requires custom `ZendeskSupportAvailabilityStrategy`. So I put `ZendeskSupportAvailabilityStrategy` to the stream definition which need customization

Reply:
Got it!